### PR TITLE
A couple of new games

### PR DESCRIPTION
**Pull request by lennyworks**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -73,71 +72,80 @@ All merged changes will be directly deployed to https://multiplayer.page!
...
+## 🎉 Party
+- [5 Second Rule*](https://5second.app/)
+- [Bad Cards*](https://bad.cards/)
+- [Charades*](https://charades.app/)
+- [Codenames](https://codenames.game/)
+- [Friendship Quiz*](https://psycatgames.com/app/friendship-quiz/)
+- [Truth or Drink*](https://truthordrink.app/)
+- [Truth or Dare*](https://psycatgames.com/app/truth-or-dare/)
+- [Skribbl*](https://skribbl.io/)
+-
```
Comment (herrherrmann) [inline readme.md]: There’s an extra dash here, which should just be an empty line instead!

```suggestion

```

**Review by herrherrmann — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution! If you get rid of the extra dash I mentioned in the inline comment, we can merge this. :)

**Comment by lennyworks** [comment]
Good catch! I've updated the pull request 😄

**Review by herrherrmann — APPROVED** [review]
Awesome, thanks for the quick fix. I’ll merge this now!

**Pull request merged**